frontend/scripts/react-components/dashboard-element/dashboard-element.container.jsx

```diff
@@ -27,9 +29,16 @@ class DashboardElementContainer extends React.Component {
   state = {
     modalOpen: true,
     goBackOnCloseModal: true,
-    step: DashboardElement.steps.WELCOME
+    step: hasVisitedBefore ? DashboardElement.steps.PANEL : DashboardElement.steps.WELCOME
   };
 
+  componentDidMount() {
+    if (!hasVisitedBefore) {
+      hasVisitedBefore = true;
+      localStorage.setItem('hasVisitedBefore', true);
```

Comment:
Finally I think a good practice is to save a date timestamp (`Date.Now()`) instead of a boolean. This allows us to reuse the same cookie in the future for users that have seen the original modal, maybe showing a new message.